tsdb/mapper.go

```diff
@@ -16,8 +16,9 @@ import (
 // aggregate output Time is ignored, and actual Time-Value pairs are contained soley
 // within the Value field.
 type MapperValue struct {
-	Time  int64       `json:"time,omitempty"`  // Ignored for aggregate output.
-	Value interface{} `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Time  int64             `json:"time,omitempty"`  // Ignored for aggregate output.
+	Value interface{}       `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Tags  map[string]string `json:"tags,omitempty"`  // Meta tags for results
```

Comment:
Attaching it there was the only place I could find that neatly allowed me to just send back the data.  We don't have access to the series cursors any longer (which have the tag knowledge).  And all the data is wired up in the `limitedRowWriter.processValues` method, which receives an slice of `MapperValues`.

https://github.com/influxdb/influxdb/pull/3628/files#diff-d6323ea51f0c01489e72144dd7198ff8R740